.releaserc.yml

```diff
@@ -0,0 +1,22 @@
+branches:
+  - main
+repositoryUrl: 'git@github.com:liatrio/envelope-game.git'
+preset: conventionalcommits
+plugins:
+  - '@semantic-release/commit-analyzer'
+  - '@semantic-release/release-notes-generator'
+  - path: '@semantic-release/github'
+    failComment: false
+    successComment: false
+    assets:
+      - path: 'envelope-*.tgz'
+        label: Helm Chart
+  - path: '@semantic-release/git'
+    assets:
+      - package.json
```

Comment:
the `package.json` isn't being updated because you aren't using the `npm` plugin.